src/components/TopBar/FileDropDown/component.tsx

```diff
@@ -52,7 +52,7 @@ export const FileDropDown = ({ cloudEnabled }: FielDropDownProps) => {
     }
 
     const authenticatedOptions = [
-        { label: "New", onClick: handleNew},
+        { label: "New Blank Project", onClick: handleNew},
```

Comment:
Does this change the current functionality? Right now on my local machine, I still get the template project.